src/schema/partner.js

```diff
@@ -247,6 +247,9 @@ const PartnerType = new GraphQLObjectType({
             partner_id: _id,
           }).then(response => {
             if (response.errors) {
+              if (response.errors.toString().match(/Document not found/)) {
```

Comment:
Also not convinced it's the best way to check for account not found error, but that seems to work:) We don't want to swallow all the errors but this specific one means that partner is not set up to accept Payments.